service/tutorial/order/NewTutorialServices.xml

```diff
@@ -0,0 +1,322 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<services xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:noNamespaceSchemaLocation="http://moqui.org/xsd/service-definition-2.1.xsd">
+    
+    <!--    Assignment 2-->
+
+    <!--    Created Order-->
+    <service verb="create" noun="OrderData" displayName="Create an order" type="inline">
```

Comment:
I think we can skip the type, as default is inline.

Reply:
updated